recipe/meta.yaml

```diff
@@ -1,26 +1,28 @@
-{% set version = "1.4.0" %}
+{% set version = "1.5.2" %}
 
 package:
   name: pkgconfig
   version: {{ version }}
 
 source:
   url: https://github.com/matze/pkgconfig/archive/v{{ version }}.tar.gz
-  sha256: 38c5de8392f4acfe7f36b25d113c496d68f534e983afdbd0ba7240c8c475b161
+  sha256: 4df0845fa1936f232b9afd951943f452acab0239de6eda753c914c199de5aab3
 
 build:
-  number: 3
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - pkg-config  # [unix]
+    - poetry
```

Comment:
Perhaps it does not need poetry in build? One less solve to worry about... at least [rich](https://github.com/conda-forge/rich-feedstock/blob/master/recipe/meta.yaml) is working without...